docs/English/Installation.md

```diff
@@ -0,0 +1,6 @@
+## Installation
```

Comment:
Like I said, configuration and contributor lists would be the only things I would consider moving to a separate file. Vital information, like how to install the software and how to use it, should be on the front page.

Reply:
Sorry, I didn't close this PR.

*I'm just continuing for fun~*

*You already said you didn't like the approach*
*but I wanted to try some working on it a bit.*

Reply:
Oh, I see. Well, knock yourself out.
That paths-ignore is a nice touch though, so I'm gonna steal it if you don't mind 😄

Reply:
Sure thing!

*Will have to remember that for other repos as well~*